### ISSUE TITLE
Sender device (Galaxy A12) provides pin with nowhere to type into.

### ISSUE THREAD

**Issue by Devnol** [grishka/NearDrop#23]
When I try to send a picture from my phone with Nearby Share, I can see my macbook just fine but when I tap on it I get a 4 digit pin I have to type in somewhere, with no prompt or anything on the macbook side. Am I missing something?

**Comment by grishka** [comment]
Yes you are. It should simply match. No need to type anything anywhere.

**Comment by Devnol** [comment]
match with what? I see no code or any prompt to receive a file on the mac side?
